controllers/helpers.go

```diff
@@ -284,17 +284,16 @@ func getImageUUID(client *nutanixClientV3.Client, imageName, imageUUID *string)
 	return foundImageUUID, nil
 }
 
-func isExistingVM(client *nutanixClientV3.Client, vmUUID string) (bool, error) {
-	vm, err := findVMByUUID(client, vmUUID)
-	if err != nil {
-		errorMsg := fmt.Errorf("error finding vm with uuid %s: %v", vmUUID, err)
-		klog.Error(errorMsg)
-		return false, errorMsg
-	}
+// func isExistingVM(client *nutanixClientV3.Client, vmUUID string) (bool, error) {
```

Comment:
Can we remove the function completely if not required?

Reply:
lets remove it from this PR. @swetangfinviya

Reply:
Removed in the latest commit